src/TrackProgress/SFA.DAS.TrackProgress/Application/Commands/TrackProgressCommand.cs

```diff
@@ -59,6 +59,13 @@ public async Task<TrackProgressResponse> Handle(TrackProgressCommand request, Ca
         if (apprenticeshipResult.Body.Apprenticeships?.Count(x => x.StartDate == request.PlannedStartDate) > 1)
             return new TrackProgressResponse(HttpStatusCode.NotFound, "Multiple results for start date");
 
+        if (request.ProviderContext.InSandboxMode)
```

Comment:
Ideally we could have cross-cutting concern that ensures that any changes made to the database are rolled back when in sandbox mode.  

I would be nervous about relying on remembering to handle sandbox mode, it's all to easy to forget, or modify an entity without realising.

Reply:
I hadn't even considered approaching the problem that way. So are we considering forwarding the IsSandbox header to the inner API? And then automatically rolling back any transaction if IsSandbox is true?